api/turing/cluster/servicebuilder/service_builder.go

```diff
@@ -136,6 +139,7 @@ func (sb *clusterSvcBuilder) NewEnricherService(
 	secretName string,
 	knativeQueueProxyResourcePercentage int,
 	userContainerLimitRequestFactor float64,
+	initialReplicas *int,
```

Comment:
The interface uses `initialScale` but the implementation uses `initialReplicas`. Similar to my comment above, I think we should make it consistent.